services/checks/checks_test.go

```diff
@@ -113,19 +115,15 @@ func TestCollectChecks(t *testing.T) {
 
 		s.collectChecks(context.Background())
 
-		mySQLChecks := s.getMySQLChecks()
-		postgreSQLChecks := s.getPostgreSQLChecks()
-		mongoDBChecks := s.getMongoDBChecks()
-		allChecks := s.GetAllChecks()
-
-		require.Len(t, mySQLChecks, 1)
-		require.Len(t, postgreSQLChecks, 1)
-		require.Len(t, mongoDBChecks, 1)
-		require.Len(t, allChecks, 3)
+		checks, err := s.GetChecks()
+		require.NoError(t, err)
+		require.Len(t, checks, 3)
```

Comment:
Let's not do that in tests. It's clear what does this number means because it's located in the assertion function. So it's really simple to read it: "we require `checks` length to be equal 3".  A separate variable will not bring any benefit here.